testing-guidance/workflow-tests/multiplayer/basic-multiplayer-workflow-tests.md

```diff
@@ -0,0 +1,31 @@
+**Multiplayer** **Testing**
+===========================
+
+These tests are designed to cover the basic functionality related to running a level from a project that is intended to be multiplayer.
+
+General Documentation for this feature set: [https://docs.o3de.org/docs/user-guide/gems/reference/multiplayer/](https://docs.o3de.org/docs/user-guide/gems/reference/multiplayer/)
+
+**Prerequisites**
+=================
+
+*   MultiplayerSample project is acquired and built ( [https://github.com/o3de/o3de-multiplayersample](https://github.com/o3de/o3de-multiplayersample) )
+    *   If using this approach, the SampleBase level can be used for most tests
+*   Alternatively: A project has been created and built that includes the Multiplayer Gem and subsequent changes have been made to enable functionality ( [https://docs.o3de.org/docs/user-guide/gems/reference/multiplayer/multiplayer-gem/configuration/](https://docs.o3de.org/docs/user-guide/gems/reference/multiplayer/multiplayer-gem/configuration/) )
+    *   If using this approach, a level should be created in the project that utilizes a spawner and network prefabs, as these will be easy identifiers for whether multiplayer connections are working as intended
```

Comment:
nit: player spawner? We want to at some point point to Gene's O3DCon workshop talk for a simple tutorial to setting up multiplayer content.